samples/21.core-bot-app-insights/src/main/java/com/microsoft/bot/sample/corebot/app/insights/DialogAndWelcomeBot.java

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.sample.corebot.app.insights;
+
+import com.codepoetics.protonpack.collectors.CompletableFutures;
```

Comment:
is this import all right?

Reply:
This is not ok

Reply:
It is identicall to the import in sample 13

Reply:
I know it's identical, but doesn't mean to be ok 😅

Reply:
apparently it's used in line 74. Maybe we should review that

Reply:
After checking, this is correct ,see another [sample](https://github.com/microsoft/botbuilder-java/blob/28fe834716f71b7c340e250bcd6e36dd53286ca9/samples/43.complex-dialog/src/main/java/com/microsoft/bot/sample/complexdialog/DialogAndWelcome.java#L6)